x-pack/solutions/security/plugins/security_solution/public/configurations/tabs/components/integrations/integrations_grid.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { lazy } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { css } from '@emotion/react';
+import type { IntegrationCardItem } from '@kbn/fleet-plugin/public';
+import { noop } from 'lodash';
+import { FACETS_MAX_WIDTH_PX, INTEGRATIONS_GRID_MAX_WIDTH_PX } from './constants';
+import { IntegrationViewFacets } from './view_facets';
+import { IntegrationsFacets } from '../../../constants';
+
+export const PackageListGrid = lazy(async () => ({
+  default: await import('@kbn/fleet-plugin/public')
+    .then((module) => module.PackageList())
+    .then((pkg) => pkg.PackageListGrid),
+}));
+
+export interface IntegrationsGridProps {
+  view: IntegrationsFacets;
+  availableIntegrations: IntegrationCardItem[];
+  installedIntegrations: IntegrationCardItem[];
+  searchTerm: string;
+  setSearchTerm: (searchTerm: string) => void;
+}
+
+export const IntegrationsGrid = React.memo<IntegrationsGridProps>(
+  ({ view, availableIntegrations, installedIntegrations, searchTerm, setSearchTerm }) => {
+    return (
+      <EuiFlexGroup direction="column" gutterSize="none">
+        <EuiSpacer size="l" />
+        <EuiFlexGroup>
+          <EuiFlexItem
+            css={css`
+              max-width: ${FACETS_MAX_WIDTH_PX}px;
+            `}
+          >
+            <IntegrationViewFacets
+              allCount={availableIntegrations.length}
+              installedCount={installedIntegrations.length}
+              selectedFacet={view}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem
+            css={css`
+              max-width: ${INTEGRATIONS_GRID_MAX_WIDTH_PX}px;
+            `}
+          >
+            <PackageListGrid
```

Comment:
Could you plase create a `integration` folder under `public/common`  and add `PackageListGrid` with `availableIntegrations` in there? (showSearchTools could coming from props so we have an option to hide it) I'd like to reuse it in the onboarding hub: https://www.figma.com/design/DYs7j4GQdAhg7aWTLI4R69/AI4DSOC?node-id=5189-152053&m=dev

Reply:
@angorayc There's nothing unique to this call that I think would warrant a component for you to also call. 

All the 'magic' comes from the hook https://github.com/elastic/kibana/pull/217905/files#diff-4695a8074467dc48e2cc4f791098d2a01aec32d88791b85a87dbe37b86f76f04R71-R73 

That's where the styling of the cards is applied